std_msgs/MultiArrayLayout layout
float64[] T_array
float64[] O_array
int32 signal